BezierSwift/Sources/Components/Badges/StatusBadge.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  StatusBadge.swift
+//
+//
+//  Created by 구본욱 on 8/9/24.
+//
+
+import SwiftUI
+
+/// StatusBadge는 Avatar와 함께 사용되며 Root(Avatar)의 상태를 아이콘이나 이미지를 사용해 직관적으로 알려줍니다.
+public struct StatusBadge: View {
+  private let configuration: StatusBadgeConfiguration
+  
+  /// 주어진 StatusBadgeConfiguration에 따른 StatusBadge 뷰를 생성합니다.
+  /// - Parameter configuration: StatusBadge의 사이즈와 아이콘 형태를 결정짓는 configuration 객체입니다.
+  public init(configuration: StatusBadgeConfiguration) {
+    self.configuration = configuration
+  }
+
+  public var body: some View {
+    ZStack(alignment: .center) {
+      Circle()
+        .fill(BezierColor.bgWhiteHighest.color)
+
+      self.icon
+        // TODO: BezierButton 병합후 View.frame(length:) 로 변경합니다. - by Finn 2024.08.09
+        .frame(width: self.configuration.iconLength, height: self.configuration.iconLength)
+    }
+    // TODO: BezierButton 병합후 View.frame(length:) 로 변경합니다. - by Finn 2024.08.09
+    .frame(width: self.configuration.length, height: self.configuration.length)
+    .compositingGroup()
+  }
+}
+
+extension StatusBadge {
+  @ViewBuilder
+  private var icon: some View {
+    if self.configuration.doNotDisturb {
+      BezierIcon.moonFilled.image
+        .foregroundColor(self.configuration.iconColor.color)
+    } else {
+      Circle()
+        .fill(self.configuration.iconColor.color)
+    }
+  }
```

Comment:
```suggestion
  private var icon: some View {
    if self.configuration.doNotDisturb {
      BezierIcon.moonFilled.image
    } else {
      Circle()
        .fill()
    }
  }
```
로 하고 외부에서 공통으로 foregroundColor 줘도 좋을 것 같아요